src/main-process/atom-window.coffee

```diff
@@ -101,10 +101,12 @@ class AtomWindow
   hasProjectPath: -> @getLoadSettings().initialPaths?.length > 0
 
   setupContextMenu: ->
-    ContextMenu = require './context-menu'
-
     @browserWindow.on 'context-menu', (menuTemplate) =>
-      new ContextMenu(menuTemplate, this)
+      @openContextMenu(menuTemplate)
+
+  openContextMenu: (menuTemplate) ->
+    ContextMenu = require './context-menu'
```

Comment:
What do you think about declaring a `ContextMenu` variable at the top of the file, and change this line so that we require `./context-menu` only once? Like:

```coffee
# at the top...
ContextMenu = null

# this line…
ContextMenu ?= require('./context-menu')
```

Not a huge deal, but should save some cycles when opening context menus.